### ISSUE TITLE
IDE0006 warning issued, but no failed task in logfile

### ISSUE THREAD

**Issue by MagnusMikkelsen** [dotnet/roslyn#12607]
**Version Used**: 14.0.25422.01 Update 3

**Steps to Reproduce**:
1. Simply open solution

**Expected Behavior**: No error or warning when opening project

**Actual Behavior**: IDE0006 Error issued

I ran the diagnostics as described on https://github.com/dotnet/roslyn/wiki/Diagnosing-Project-System-Build-Errors
But there was no "FAILED" task.

That is why I submit this issue.

Solutionfile/contents provided on request

[companyname.TeststandHelper_Compile_9aef4df6.designtime.log.zip](https://github.com/dotnet/roslyn/files/371564/companyname.TeststandHelper_Compile_9aef4df6.designtime.log.zip)

**Comment by PhatWrat** [comment]
Same issue.
Microsoft Visual Studio Professional 2015 Version 14.0.25425.01 Update 3
Microsoft .NET Framework Version 4.6.01055

Note that VS is NOT showing "Miscellaneous Files" as noted on linked diagnostics page, but IS issuing IDE0006.  Also, there are no errors or warnings in the log file that is generated (attached)

[InStock.MasterDataInterface_Compile_c379eb77.designtime.zip](https://github.com/dotnet/roslyn/files/460276/InStock.MasterDataInterface_Compile_c379eb77.designtime.zip)

**Comment by tarminfr** [comment]
Same issue.
Microsoft Visual Studio 2015 Version 14.0.25420.1.

**Comment by Pilchie** [comment]
Sorry for the delay in looking here.  For @MagnusMikkelsen's log, the interesting bit is:
```
Target "CoreCompile" in file "C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.CSharp.Core.targets":
  Building target "CoreCompile" completely.
  Output file "__NonExistentSubDir__\__NonExistentFile__" does not exist.
  Using "Csc" task from assembly "C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.Build.Tasks.CodeAnalysis.dll".
  Task "Csc"
    The IDE's in-process compiler does not support the specified values for the "LangVersion" parameter.  Therefore, this task will fallback to using the command-line compiler.
  Done executing task "Csc".
  Task "CallTarget" skipped, due to false condition; ('$(TargetsTriggeredByCompilation)' != '') was evaluated as ('' != '').
Done building target "CoreCompile" in project "companyname.TeststandHelper.csproj".
```

Can you share what `LangVersion` is set to in your .csproj?

**Comment by Pilchie** [comment]
@PhatWrat - Same thing on yours.

**Comment by Pilchie** [comment]
@heejaechang @jasonmalinowski looks like we might need to update the logic that triggers IDE0006?

**Comment by heejaechang** [comment]
current logic is basically this.

http://torch/QuickView.ashx?f=src%5cvsproject%5clangbuild%5clangcompiler.cpp&q=setintellisensebuildresult&index=Dev15_15.6.25605.1#1320

if RunIntellisenseBuildInternal fails, then we show the error. @jasonmalinowski is there a way to get specific reason why the call failed? I don't remember all the details but when I looked, there was no straight forward way to retrieve the reason the call failed.

my hacky way to retrieve actual log file name which cause the call to fail was all I could do. and I believe it couldn't be 100% correct so you guys want me to take that out.

**Comment by heejaechang** [comment]
I think the best bet is waiting for CPS since I believe the new CPS project system will put design time errors in error list.

tagging @srivatsn @davkean

**Comment by jasonmalinowski** [comment]
@Pilchie That error you're seeing about LangVersion is probably this hack still active:

https://github.com/dotnet/roslyn/blob/4d6e95517d1b29d5028340da656ed47815b07f87/src/VisualStudio/CSharp/Impl/ProjectSystemShim/CSharpProjectShim.ICSCompilerConfig.cs#L48-L56

Ages ago we had a bug tracking cleaning this up; no idea where it went.

**Comment by jasonmalinowski** [comment]
@heejaechang Without threading the data through, not easily.

**Comment by MagnusMikkelsen** [comment]
@Pilchie LangVersion tag is not present in the .csproj file - in the gui it says Language Version: default

**Comment by kamenlitchev** [comment]
Hello, I encounter the same thing. No errors in build log. Tried setting the LangVersion but same thing happens.

[Yyy.Zzz.UWP_Compile_422e8aed.designtime.txt.zip](https://github.com/dotnet/roslyn/files/1343808/Yyy.Zzz.UWP_Compile_422e8aed.designtime.txt.zip)

**Comment by CyrusNajmabadi** [comment]
Have heard no more reports on this, and i haven't seen this in years.  closing out.
